assets/helpers/checkouts.js

```diff
@@ -72,7 +73,15 @@ function getValidContributionTypesFromUrlOrElse(fallback: ContributionType[]): C
   return fallback;
 }
 
-function getValidContributionTypes(): ContributionType[] {
+function getValidContributionTypes(abParticipations: Participations): ContributionType[] {
+  const { globalContributionTypes } = abParticipations;
+  const params = globalContributionTypes
+    ? globalContributionTypes.split('_')
```

Comment:
can this bit just be `if (globalContributionTypes === 'default-annual_no-monthly') return ['ONE_OFF', 'ANNUAL']`?

Reply:
👍